draft-ietf-acme-acme.md

```diff
@@ -160,33 +160,31 @@ from the client.
 # Protocol Overview
 
 ACME allows a client to request certificate management actions using a set of
-JSON messages carried over HTTPS.   In some ways, ACME functions much like a
-traditional CA, in which a user creates an account, adds identifiers to that
-account (proving control of the domains), and requests certificate issuance for
-those domains while logged in to the account.
+JSON messages carried over HTTPS.   In many ways, ACME functions much like a
+traditional CA, in which a user creates an account, requests a certificate,
+and proves control of the domains in that certificate in order for the CA to
+sign the requested certificate.
 
-In ACME, the account is represented by an account key pair.  The "add a domain"
```

Comment:
Or rather, "In ACME, there are accounts" is redundant with the next paragraph:

> The first phase of ACME is for the client to register with the ACME server. The client generates an asymmetric key pair and associates this key pair with a set of contact information by signing the contact information. The server acknowledges the registration by replying with a account object echoing the client's inputrequest an account with the ACME server. The client generates an asymmetric key pair and requests a new account, optionally providing contact information, agreeing to terms of service, and/or associating the account with an existing account in another system. The creation request is signed with the generated private key to prove that the client controls it.

I think that paragraph does an adequate job describing how ACME accounts are similar to traditional CA accounts. Do you disagree?